lms/djangoapps/instructor/enrollment.py

```diff
@@ -259,14 +264,19 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
             module_state_key.to_deprecated_string(),
         )
 
-    module_to_reset = StudentModule.objects.get(
-        student_id=student.id,
-        course_id=course_id,
-        module_state_key=module_state_key
-    )
+    try:
+        module_to_reset = StudentModule.objects.get(
+            student_id=student.id,
+            course_id=course_id,
+            module_state_key=module_state_key
+        )
+    except StudentModule.DoesNotExist, ex:
```

Comment:
Nit: The `ExceptionClass, exception_obj` syntax for except statements is deprecated. Use `except StudentModule.DoesNotExist as ex` instead.